presto-main/src/main/java/com/facebook/presto/type/VarcharOperators.java

```diff
@@ -41,58 +41,58 @@ private VarcharOperators()
     {
     }
 
-    @LiteralParameters({"x", "y"})
+    @LiteralParameters({"x"})
```

Comment:
These should use the simple form since they only have one arg:
```java
 @LiteralParameters("x")
```
There's an IntelliJ warning for this.